src/platform/export/fileConverter.ts

```diff
@@ -107,10 +108,17 @@ export class FileConverter implements IFileConverter {
         }
 
         if (target) {
-            await this.exportFileOpener.openFile(format, target, true);
```

Comment:
Moved this code into a different method so that the child classes can do what they like (change behaviour on how files are opened after its exported)